tools/schemacode/bidsschematools/tests/test_validator.py

```diff
@@ -362,3 +364,23 @@ def test_broken_json_dataset(bids_examples, tmp_path):
         dataset_path,
         report_path=True,
     )
+
+
+def test_error_datasets(bids_error_examples):
+    from bidsschematools.validator import validate_bids
+
+    schema_path = "{module_path}/data/schema/"
+
+    for i in os.listdir(bids_error_examples):
```

Comment:
Ideally (so we see on which example it fails and which not instead of failing on the first one) this looping should be done via `@pytest.mark.parametrize` over this test but I am not sure if that is feasible to merry with `bids_error_examples` fixture.  @jwodder -- do you know if that is possible?

Reply:
What about:

```Python
from bidsschematools.conftest import BIDS_ERROR_SELECTION

@pytest.mark.parametrize("dataset", BIDS_ERROR_SELECTION)
def test_error_datasets(bids_error_examples, dataset):
    ...
```

Reply:
@effigies *Iff* `BIDS_ERROR_SELECTION` equals the set of non-dot entries in `bids_error_examples`, then that would work.

Reply:
short term this `BIDS_ERROR_SELECTION` should work but IMHO there should be no "whitelisting" here -- all metadata pertinent to what datasets are good or bad should be within those respective example repositories. That would lead for those `*_SELECTION` variables to disappear and thus this solution no longer work.  But I guess for now it is better than `for` loop indeed ;)